pkg/helm/actions/install_chart_test.go

```diff
@@ -1 +1,33 @@
 package actions
+
+import (
+       "github.com/stretchr/testify/assert"
+       "helm.sh/helm/v3/pkg/action"
+       "helm.sh/helm/v3/pkg/chartutil"
+       kubefake "helm.sh/helm/v3/pkg/kube/fake"
+       "helm.sh/helm/v3/pkg/storage"
+       "helm.sh/helm/v3/pkg/storage/driver"
+       "io/ioutil"
+       "testing"
+)
+
+func TestInstallChart(t *testing.T) {
+	       chart := "../testdata/influxdb-3.0.2.tgz"
+	       store := storage.Init(driver.NewMemory())
+	       actionConfig := &action.Configuration{
+		               Releases:     store,
+		               KubeClient:   &kubefake.PrintingKubeClient{Out: ioutil.Discard},
+		               Capabilities: chartutil.DefaultCapabilities,
+		               Log:          func(format string, v ...interface{}) {},
+		       }
+		       act := NewHelmActions(actionConfig)
+	       _, err := act.InstallChart("test", "test", chart, nil)
+	       if err != nil {
+		               t.Error(err)
+		       }
+	       rels, err := act.ListReleases()
+	       if err != nil {
+		               t.Error(err)
+		       }
+	       assert.Len(t, rels, 1, "Release list should contain installed chart release")
```

Comment:
asserting just the number of returned releases is not enough, i.e. there could be a situation that the installed release does not correspond to the input params. Thus, please check if release name is equal to one submitted in line 24, and check other important release attributes. Also, please check if `InstallChart` returns what contract specifies. There is no need to use `ListReleases()` in this test, because if something goes wrong, you are not sure if this is so because `ListReleases()` has a bug, or the issue is in `InstallChart()`. Please fix that.

Reply:
Done